src/main/java/seedu/address/model/person/Donor.java

```diff
@@ -51,17 +50,12 @@ public boolean equals(Object other) {
         }
 
         // instanceof handles nulls
-        if (!(other instanceof seedu.address.model.person.Donor)) {
+        if (!(other instanceof Donor)) {
             return false;
         }
 
-        seedu.address.model.person.Donor otherDonor = (seedu.address.model.person.Donor) other;
-        return name.equals(otherDonor.name)
-                && phone.equals(otherDonor.phone)
-                && email.equals(otherDonor.email)
-                && address.equals(otherDonor.address)
-                && tags.equals(otherDonor.tags)
-                && donatedAmount.equals(otherDonor.donatedAmount);
+        seedu.address.model.person.Donor otherDonor = (Donor) other;
```

Comment:
I think we can import it at the top! Will allow for code to look cleaner 😃

Reply:
Oops, I missed this part. Fixed it!